typescript/src/client/evm/evm.types.ts

```diff
@@ -423,6 +425,20 @@ export interface UpdateEvmAccountOptions {
   idempotencyKey?: string;
 }
 
+/**
+ * Options for updating an EVM smart account.
+ */
+export interface UpdateEvmSmartAccountOptions {
+  /** The address of the account. */
+  address: string;
```

Comment:
The `Address` type has the `0x`-prefix built-in, providing a better DevX!
```suggestion
  address: Address;
```